master/static/migrations/20200616205220_extend-steps-table.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE public.steps ADD COLUMN num_batches integer NULL;
```

Comment:
question: what happens if you have allow this field to default to NULL, but it's an `int` (not an `int*`) in `model.Step`?  Does  that break the master's ability to read old steps from the database, like if users were going to run `det t describe` on a trial from before this change?

Reply:
Valid concern, sqlx (or something below it) seems to handle it just fine.